modules/storages/app/models/storages/storage_file.rb

```diff
@@ -0,0 +1,15 @@
+class Storages::StorageFile
+  attr_reader :id, :name, :size, :mime_type, :created_at, :last_modified_at, :created_by_name, :last_modified_by_name, :location
+
+  def initialize(id, name, size, mime_type, created_at, last_modified_at, created_by_name, last_modified_by_name, location)
```

Comment:
Idealy this class would have value semantics and be immutable and not created by calling the `ctor` directly. I should have a validating static create method that raises or chooses sane defaults for the resulting object. But this can be done easily yes.

Reply:
The `Struct` is exactly the right thing for this value object. Thanks for sharing.

Reply:
I reverted the change in discussion with @ulferts, because `Struct` is not suitable for this case. It makes the code more complicated.